static/templates/settings/organization_permissions_admin.hbs

```diff
@@ -167,7 +167,7 @@
                         </label>
                         <input type="text" id="id_realm_message_content_edit_limit_minutes"
                           name="realm_message_content_edit_limit_minutes"
-                          class="admin-realm-time-limit-input prop-element"
```

Comment:
Why is `prop-element` removed for these inputs? I forgot what exactly that does.

Reply:
Oh, clarified in the commit message, makes sense!